examples/dreambooth/README_flux.md

```diff
@@ -0,0 +1,153 @@
+# DreamBooth training example for FLUX.1 [dev]
+
+[DreamBooth](https://arxiv.org/abs/2208.12242) is a method to personalize text2image models like stable diffusion given just a few (3~5) images of a subject.
+
+The `train_dreambooth_flux.py` script shows how to implement the training procedure and adapt it for [FLUX.1 [dev]](https://blackforestlabs.ai/announcing-black-forest-labs/). We also provide a LoRA implementation in the `train_dreambooth_lora_flux.py` script.
```

Comment:
Let's also add a disclaimer about the high memory requirements here. And refer people to @bghira's guide in case people are interested to try it out on a resource-constrained device.